pages/index.vue

```diff
@@ -74,24 +75,15 @@
         }
       }
     },
-    //middleware: 'notAuthenticated',
     methods: {
-      async login() {
+      async doLogin() {
         try {
-
-          await this.$auth.loginWith('local', {
-            data: {
-              "email": this.$v.form.email.$model,
-              "password": this.$v.form.password.$model
-            }
-          })
-          
-
-            if (this.$auth.loggedIn) {
-              this.$router.push('/home')
-            }
+          this.$router.push(await login({
+            auth: this.$auth, 
+            email: this.$v.form.email.$model , 
+            password: this.$v.form.password.$model}))
         } catch (e) {
-          console.log('Username or Password wrong')
+          console.log(e)
```

Comment:
console.log to replace with a visual notification for user experience.
suggestion: you can use bootstrap notification